node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I also have no idea what this file is, but it's probably specific to your machine.